_search-plugins/querqy/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: Querqy plugin
+has_children: false
+redirect_from:
+  - /search-plugins/querqy/
+nav_order: 10
+---
+
+# Querqy plugin
+
```

Comment:
Include a one-sentence overview of what the Quergy plugin is and does. The heading is sort of orphaned without a paragraph.

Reply:
made suggested change